efficacy/results.md

```diff
@@ -28,7 +28,7 @@ See the [Fair Efficacy Paper page]({{"/efficacy/paper" | relative_url }}) for de
 
 |Protocol|Latest results|Test reports|
 |---|---|---|
-|<b>Herald Test App</b><br>(Herald Protocol)<br>(Tolerates 1 of 2 phones without advertising)|<b>Efficacy: 41.37%<br>(Effective)</b><br>Pspec^2: 48.71%<br>Detection: 100%<br>Continuity: 99.991%<br>Completeness: 98.36%<br>Accuracy: 95.95%|Dev team tests: <br />v1.1 Release: [2020-11-28]({{"/results/herald-2020-11-28" | relative_url }})<br />v1.2.0-beta2 in development: [2020-12-08]({{"/results/herald-2020-12-08" | relative_url }})<br>See the bottom of this page for historical Herald results. |
+|<b>Herald Test App</b><br>(Herald Protocol)<br>(Tolerates 1 of 2 phones without advertising)|<b>Efficacy: 41.37%<br>(Effective)</b><br>Pspec^2: 48.71%<br>Detection: 100%<br>Continuity: 99.991%<br>Completeness: 98.36%<br>Accuracy: 95.95%|Dev team tests: <br />v1.1 Release: [2020-11-28]({{"/results/herald-2020-11-28" | relative_url }})<br />v1.2.0-beta2 in development: [2020-12-08]({{"/results/herald-2020-12-08" | relative_url }})<br />v1.2.0-beta3 in development: [2021-01-31]({{"/results/herald-2021-01-31" | relative_url }})<br>See the bottom of this page for historical Herald results. |
```

Comment:
Please remove v1.2.0-beta2 from this summary. Only the latest release and develop should be shown on this table.

Reply:
Updated accordingly.